CreateAnExperiment.md

```diff
@@ -5,7 +5,7 @@ Experiments should be contained within a branch in the dotnet/runtimelab reposit
 ## Steps to setup a new experiment
 
 - Pick a good name for your experiment and create branch for it in dotnet/runtimelab. Branch names should be prefixed with `feature/` in order to have official build support.
-   - If the experiment is expected to require changes of .NET runtime itself, it should be branched off of [dotnet/runtimelab:runtime-master](https://github.com/dotnet/runtimelab/tree/runtime-master) that is a automatically maintained mirror of [dotnet/runtime:master](https://github.com/dotnet/runtime/tree/master).
+   - If the experiment is expected to require changes of .NET runtime itself, it should be branched off of [dotnet/runtimelab:runtime-master](https://github.com/dotnet/runtimelab/tree/runtime-master) that is a automatically maintained mirror of [dotnet/runtime:master](https://github.com/dotnet/runtime/tree/main).
```

Comment:
Are we also going to rename `dotnet/runtimelab:runtime-master` to `dotnet/runtimelab:runtime-main` ?

Reply:
I was going to handle that as we also need to update the mirror.

Reply:
Yes, that will be part of the runtimelab repo M2M renaming.